alienfile

```diff
@@ -0,0 +1,41 @@
+use alienfile;
+use Config;
+
+configure { requires 'Alien::Build::Plugin::Build::Make' => '0.01' };
+
+plugin 'Probe::Vcpkg' => (
+    name     => 'libsodium',
+    ffi_name => 'libsodium',
+);
+plugin 'PkgConfig' => (
+    pkg_name => 'libsodium',
+    minimum_version => '1.0.3',
+);
+
+share {
+    my $run_upstream_tests = ($ENV{AUTOMATED_TESTING} || $ENV{RELEASE_TESTING}) ? 1 : 0;
+
+    # note on apple weirdisms: https://github.com/joyent/libuv/issues/1200
+    meta->prop->{env}->{LIBTOOLIZE} = 'libtoolize' if $^O eq 'darwin';
+
+    plugin Download => (
+        url     => 'https://download.libsodium.org/libsodium/releases/',
+        version => qr/^libsodium-([0-9.]+)-stable\.tar\.gz$/,
+    );
+
+    plugin Extract => 'tar.gz';
+
+    requires 'Alien::Autotools';
+    plugin 'Build::Autoconf' => ();
+
+    build [
+        'sh autogen.sh',
```

Comment:
I don't think you need this (or `Alien::Autotools`) as it looks like the tarball comes with a `./configure`.

Reply:
Looks like removing this somehow removed a prereq on Sort::Versions. I'll get that added in specifically.